pages/linux/brightnessctl.md

```diff
@@ -0,0 +1,24 @@
+# brightnessctl
+
+> Utility for reading and controlling device brightness for GNU/Linux operating systems.
+> More information: <https://github.com/Hummer12007/brightnessctl>.
+
+- List devices with available brightness control:
+
+`brightnessctl --list`
+
+- Get the currently set brightness of the display backlight:
+
+`brightnessctl get`
+
+- Set the brightness of the display backlight to a percenage of the maximum supported value:
+
+`brightnessctl set {{50%}}`
+
+- Increase brightness by certain percent:
+
+`brightnessctl set {{+10%}}`
+
+- Decrease brightness by certain percent:
```

Comment:
```suggestion
- Decrease brightness by a specified decrement:
```

Reply:
I do not like 'decrease and 'decrement' in the same sentence, maybe the following is better:
`- Decrease brightness by a specified amount:`

Reply:
I don't personally see the issue, since ` - Increase __________ increment` was just applied, which in return is just the complete opposite of each other completely, since the words chosen are antonyms